python/lsst/afw/table/schema/schema.cc

```diff
@@ -0,0 +1,522 @@
+    // not true enums
+    cls.attr("EQUAL_KEYS") = py::cast(int(Schema::EQUAL_KEYS));
+    cls.attr("EQUAL_NAMES") = py::cast(int(Schema::EQUAL_NAMES));
+    cls.attr("EQUAL_DOCS") = py::cast(int(Schema::EQUAL_DOCS));
+    cls.attr("EQUAL_UNITS") = py::cast(int(Schema::EQUAL_UNITS));
+    cls.attr("EQUAL_FIELDS") = py::cast(int(Schema::EQUAL_FIELDS));
+    cls.attr("EQUAL_ALIASES") = py::cast(int(Schema::EQUAL_ALIASES));
+    cls.attr("IDENTICAL") = py::cast(int(Schema::IDENTICAL));
+
+    cls.def(py::init<>());
+    cls.def(py::init<Schema const &>());
+    cls.def("__getitem__", [](Schema & self, std::string const & name) { return self[name]; });
+    cls.def("__eq__",
+           [](Schema const & self, Schema const & other) { return self == other; },
+           py::is_operator());
+    cls.def("__ne__",
+           [](Schema const & self, Schema const & other) { return self != other; },
+           py::is_operator());
+    cls.def("getCitizen", &Schema::getCitizen, py::return_value_policy::reference_internal);
+    cls.def("getRecordSize", &Schema::getRecordSize);
+    cls.def("getFieldCount", &Schema::getFieldCount);
+    cls.def("getFlagFieldCount", &Schema::getFlagFieldCount);
+    cls.def("getNonFlagFieldCount", &Schema::getNonFlagFieldCount);
+    cls.def(
+        "find",
+        [](py::object const & self, py::object const & key) -> py::object {
+            try {
+                if (py::isinstance<py::str>(key) || py::isinstance<py::bytes>(key)) {
+                    Schema const & s = py::cast<Schema const &>(self);
+                    std::string name = py::cast<std::string>(key);
+                    MakePythonSchemaItem func;
+                    s.find(name, func);
+                    return func.result;
+                }
+                return key.attr("_findIn")(self);
+            } catch (pex::exceptions::NotFoundError & err) {
+                // Avoid API change by re-throwing as KeyError.
+                PyErr_SetString(PyExc_KeyError, err.what());
+                throw py::error_already_set();
+            }
+        }
+    );
+    cls.def("getNames", &Schema::getNames, "topOnly"_a=false);
+    cls.def("getAliasMap", &Schema::getAliasMap);
+    cls.def("setAliasMap", &Schema::setAliasMap, "aliases"_a);
+    cls.def("disconnectAliases", &Schema::disconnectAliases);
+    cls.def("forEach", [](Schema & self, py::object & obj) { self.forEach(obj); });
+    cls.def("compare", &Schema::compare, "other"_a, "flags"_a=int(Schema::EQUAL_KEYS));
+    cls.def("contains", (int (Schema::*)(Schema const &, int) const) &Schema::contains,
+            "other"_a, "flags"_a=int(Schema::EQUAL_KEYS));
+    cls.def(
+        "__contains__",
+        [](py::object const & self, py::object const & key) {
+            try {
+                self.attr("find")(key);
+            } catch (py::error_already_set & err) {
+                return false;
+            }
+            return true;
+        }
+    );
```

Comment:
Can the `__contains__` method match the `contains` method? I think that would better match user expectations (especially if we also deprecate `contains`)